presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -3233,6 +3233,24 @@ public void testShowCreateTable()
         assertUpdate(createTableSql);
         actualResult = computeActual("SHOW CREATE TABLE \"test_show_create_table'2\"");
         assertEquals(getOnlyElement(actualResult.getOnlyColumnAsSet()), createTableSql);
+
+        createTableSql = format("" +
+                        "CREATE TABLE %s.%s.%s (\n" +
+                        "   c1 ROW(\"$a\" bigint, \"catalog\" varchar)\n)\n",
+                getSession().getCatalog().get(),
+                getSession().getSchema().get(),
+                "test_show_create_table_with_special_characters");
+        assertUpdate(createTableSql);
+        assertThat(computeActual("SHOW CREATE TABLE test_show_create_table_with_special_characters").getOnlyValue())
+                .isEqualTo(format("" +
+                                "CREATE TABLE %s.%s.%s (\n" +
+                                "   c1 ROW(\"$a\" bigint, catalog varchar)\n)\n",
+                        getSession().getCatalog().get(),
+                        getSession().getSchema().get(),
+                        "test_show_create_table_with_special_characters") +
+                        "WITH (\n" +
+                        "   format = 'ORC'\n" +
+                        ")");
```

Comment:
you should be able to use `createTableSql` here
you cannot because `catalog` is no longer delimited

but, that also means  user cannot take `SHOW CREATE TABLE` output and use it to create the table.
Since this is _the purpose_ of the SHOW CREATE TABLE query, it's  a no-go to me.

Reply:
logged https://github.com/prestosql/presto/issues/3584